src/ebay-video/__tests__/index.stories.tsx

```diff
@@ -1,18 +1,21 @@
 import React, { useState } from 'react'
 import { Meta } from '@storybook/react'
-import { EbayButton, EbayVideo, EbayVideoSource } from '../../index'
+import 'shaka-player/dist/controls.css'
```

Comment:
We need to add this info in the readme also that applications needs to import this css

Reply:
added